core/urls.py

```diff
@@ -49,6 +49,7 @@
                 # v1 urls
                 path("rester/", include("rester.urls")),
```

Comment:
Remove the applications of the template as it doesn't have any functionality in the code source